src/coreclr/vm/unsafeaccessors.cpp

```diff
@@ -311,7 +594,7 @@ namespace
 
         // Following a similar iteration pattern found in MemberLoader::FindMethod().
         // However, we are only operating on the current type not walking the type hierarchy.
-        MethodTable::IntroducedMethodIterator iter(pMT);
+        MethodTable::IntroducedMethodIterator iter(pMT, /* restrictToCanonicalTypes */ FALSE);
```

Comment:
```suggestion
        MethodTable::IntroducedMethodIterator iter(pMT);
```
Is this left over from previous iterations or is this still needed?